staging/src/k8s.io/apiserver/pkg/server/options/etcd.go

```diff
@@ -151,6 +152,10 @@ func (s *EtcdOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.StringSliceVar(&s.StorageConfig.ServerList, "etcd-servers", s.StorageConfig.ServerList,
 		"List of etcd servers to connect with (scheme://ip:port), comma separated.")
 
+	fs.DurationVar(&s.StorageConfig.AutoSyncInterval, "etcd-auto-sync-interval", s.StorageConfig.AutoSyncInterval, ""+
+		"etcd-auto-sync-interval is the interval to update etcd endpoints with its latest members. 0 for disabling auto sync, default value is 0. "+
```

Comment:
"The interval at which to update etcd endpoints to the latest members as reported by the cluster. Do not enable if the api server has a different network view of the etcd cluster members than the etcd cluster reports (for example, if accessing etcd via a load-balancer). Disabled when set to 0."

Reply:
ok